src/hooks/useNavigation.js

```diff
@@ -4,10 +4,18 @@ import { useSoftkey } from 'hooks'
 export const useNavigation = (origin, containerRef, axis, elementsSelector = '[data-selectable]') => {
   const [current, setCurrent] = useState({ type: null, index: 0, key: null })
 
-  const getAllElements = () => document.querySelectorAll(elementsSelector)
+  const getAllElements = () => {
+    if (containerRef.current === undefined) {
```

Comment:
I would like to understand when is `containerRef.current` undefined. I feel like querying `document` is never OK but I might be missing some context here.

Reply:
`containerRef.current` is undefined when the app first loads the `Search` home screen. If we comment the if-statement here in line 8, this is stack trace that the console throws:

<img width="812" alt="image" src="https://user-images.githubusercontent.com/4752599/79898871-f3399300-83d9-11ea-8fc9-c5bd374eec1d.png">

It’s from the `setNavigation` in the component's `useEffect`. This error messes up the loading of `Search` because softkeys don’t get set, the cursor doesn’t get set, and querying doesn’t work. 

I am returning `document.querySelectorAll(elementsSelector)` in this case because querying `document` is what we were doing previously.

Reply:
`useEffect` runs after the component has rendered and the refs are populated. However, in this case the ref is on the results list which doesn't exist until there are search results. What we are trying to do here with `setNavigation(0)` is to focus the input box. 

Have 2 refs would fix that too. Or we could be more explicit with something along the lines of `inputRef.current.focus()`